contents/blog/making-something-people-want.md

```diff
@@ -0,0 +1,149 @@
+
+This made us feel very close with users.
+
+We realized we had to _act_ on feedback/bugs and feature requests. We knew that if we acted on them, we'd get more, which means we'd make more progress. If we didn't do this, then users would stop giving us feedback and we'd die!
+
+4. Repeat
+
+For every new feature, we aimed to get a few early adopter users on it, and through the above loop. Every time we did so, this led to bug reports, UX improvements or something totally different. A bit like QA-ing a pull request.
+
+It is _much _ better to ship like this, rather than shipping in isolation. With this approach, what you build will be much more likely to work, and you'll gain a far deeper understanding of your customers too.
+
+This _still_ applies. See our [community roadmap idea](https://github.com/PostHog/posthog.com/issues/4453) - we're looking to go further with this.
+
+## How to engage with users
+
+Once we launched, we made it _very_ easy to give us feedback. Slack turned out to be the best channel because:
+
+* many users had it open all day at work - we saw 20x the response rate of email
+* there's a low bar to posting because people use it informally - people don't spend a lot of time to polish their communication on it. We wanted more communication, rather than less, especially in the early days
+
+There are some issues with using Slack like this:
+
+* It isn't ticketed (without looking for 3rd party apps) - this can be painful now we've several thousand people in there
+* Once you have lots of people in your group, you lose messages if you don't pay for their premium features (which are way too expensive for most free communities to use)
+
+(Later on, we built [Squeak!](https://github.com/posthog/squeak) to solve some of these)
+
+## Enter the Ideal Customer Profile (ICP) - focus on specific types of user
+
+We spent the first 18 months focused on the open source project. We made a mistake though - we didn't get specific about who we were building for. We'd have said "developers are all the same, aren't they?"
+
+Bluntly, defining an ICP felt bureaucratic and like something a Fortune 500 would do. I put it in my mental bucket of things I don't care about, alongside words such as "synergy", "resources" and "alignment".
+
+Alas, I was wrong. I think we could have saved about 30% of our time if we'd done this sooner.
+
+This is something I'd do from day 1 if I were starting over.
+
+Something we only cracked once we focused on our paid product was the importance of defining an Ideal Customer Profile. It's _still_ a work in progress.
+
+Broadly, create a very, *very* specific list of characteristics and behaviors of the users you are building for. Look at [PostHog's current ICP](/handbook/strategy/overview#target-customers-for-2022) if you want an example.
+
+## Listen to users
```

Comment:
Maybe integrate with "How to engage with users" section?

Reply:
I can't otherwise work out how to conclude!